bin/wedding_cake.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+from scipy.special import logsumexp
+from numpy.testing import assert_array_less
+from anesthetic import NestedSamples
+import matplotlib.pyplot as plt
+
+np.random.seed(0)
+D=4
+alpha = 0.5
+sigma = 0.01
+nlive = 500
+
+i = np.arange(1000)
+ri = alpha**(i/D)/2
+logZ = logsumexp(-ri**2/2/sigma**2 + i*np.log(alpha) + np.log(1-alpha))   
+
+def r(x):
+    return np.max(abs(x-0.5),axis=-1)
+
+def i(x):
+    return np.floor(D*np.log(2*r(x))/np.log(alpha)) 
+
+def logL(x):
+    ri = alpha**(i(x)/D)/2
+    return - ri**2/2/sigma**2
+
+
```

Comment:
you may want to put some of this in a main guard, if __name__ == '__main__': ...? then the weddingcake function can potentially be imported for use elsewhere w/o running the computations etc here

also seems like repetition of the test/wedding_cake.py?

Reply:
You're right that it's a repetition of the code in test/wedding_cake.py. As the wedding cake example can be generated efficiently on-demand in memory, I've removed the repeated bin/wedding_cake.py code, as well as the text file example. It would probably be good in future versions of anesthetic to integrate more examples of this kind of test